scripts/generate-icon-react-wrapper.js

```diff
@@ -11,7 +11,85 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { generateIconWrapper } from './cem-plugin-react-wrapper.js';
+import { readFile } from 'fs/promises';
+import fsExtra from 'fs-extra';
+import { basename, dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+import prettier from 'prettier';
+import Case from 'case';
+import fg from 'fast-glob';
+import yaml from 'js-yaml';
+
+const { existsSync, outputFile, readFileSync, readJSON } = fsExtra;
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const prettierConfig = yaml.load(
+    readFileSync(resolve(__dirname, '..', '.prettierrc.yaml'))
+);
+
+/**
+ * Core entry function
+ */
+export async function generateIconWrapper(iconType) {
```

Comment:
This is failing, we either need to move this back to `cem-plugin-react-wrapper` or we need to export `genIconReactComponent`, `genIconNextComponent`, `genPackageJson`, `genTsconfigJson`.

Reply:
Still failing, ignore outdated flag